build.sbt

```diff
@@ -38,7 +38,8 @@ libraryDependencies ++= Seq(
   "dev.zio"      %% "zio-nio"      % "1.0.0-RC9",
   "dev.zio"      %% "zio-test"     % zioVersion % "test",
   "dev.zio"      %% "zio-test-sbt" % zioVersion % "test",
-  "org.polynote" %% "uzhttp"       % "0.2.6"    % "test"
+  "org.polynote" %% "uzhttp"       % "0.2.6"    % "test",
+  "dev.zio"      %% "zio-json"     % "0.1"
```

Comment:
I think we probably don't want to add a `zio-json` dependency here. In any other project would be fine but we are really trying to say this is zero dependency other than ZIO. We even want to rip `zio-nio` out. Can we write our own encoders for these data types? I think it should be a very simple hierarchy.

Reply:
Sure I can do it. It is just more error prone, but probably fine. I can add `zio-json` or `circe` to test dependencies and compare results from manual and derived encoding.

Reply:
That's a good idea to compare to `zio-json` in the tests.